notebooks/Tutorial1_A_basic_avatarization.ipynb

```diff
@@ -467,6 +467,42 @@
     "The report is now generated and available on your machine"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "30d6f795",
+   "metadata": {},
+   "source": [
+    "# How to print an error message \n",
+    "There are multiple types of error and we encourage you to have a look at our [documentation](https://python.docs.octopize.io/latest/user_guide.html#understanding-errors) to understand them.\n",
+    "\n",
+    "The most common error is when server validation prevents a job from running.\n",
+    "\n",
+    "The following section show how to print an error message. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bbe6d5ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wrong_parameters = AvatarizationParameters(\n",
+    "    k=500, dataset_id=dataset.id\n",
```

Comment:
you can use len(df) instead to make sure this will fail no matter the fixture used in input